code/tests/files/torch/torch.q

```diff
@@ -1,13 +1,16 @@
+// code/tests/files/torch/torch.q - PyTorch test files
+// Copyright (c) 2021 Kx Systems Inc
+
 \d .automl
 
 // @kind function
 // @category models
-// @fileoverview Fit model on training data and score using test data
-// @param data  {dict} containing training and testing data according to keys
+// @desc Fit model on training data and score using test data
+// @param data  {dictionary} containing training and testing data according to keys
```

Comment:
All unnecessary spaces can be removed for the file, and start of each description can be capitalized